unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/TriggerArea.cs

```diff
@@ -25,6 +28,10 @@ public override HashSet<GameObject> DetectAvatars(Vector3 center, Quaternion rot
         HashSet<GameObject> result = new HashSet<GameObject>();
         foreach (Collider collider in colliders)
         {
+            if (excludeColliders != null && excludeColliders.Contains(collider))
```

Comment:
since null checks can be a bit costly, can we cache the `excludeColliders != null` in a bool just before the foreach? and use that bool inside the foreach, what do you think?

Reply:
👍 Changed here: https://github.com/decentraland/unity-renderer/pull/1853/commits/343a0512997ef9de3670de0da08f71f8c6a0780b